src/getTime.js

```diff
@@ -18,7 +18,13 @@
  * @returns {string} - valid time
  */
 function getTime(str) {
-  // write code here
+  const regStr = /\b([0-1][0-9]|[2][0-3]):[0-5][0-9]\b/;
+
+  if (regStr.test(str)) {
+    return str.match(regStr)[0];
+  } else {
+    return '';
+  }
```

Comment:
use the ternary operator (google if you do not know) (optional for your solution)